### ISSUE TITLE
Preffered Sans/Serif Font

### ISSUE THREAD

**Issue by alijan92** [bohoomil/fontconfig-ultimate#116]
Hello, bohoomil!

First of all, I want to thank you for your contributing!

I use fontconfig-ultimate with 'free' preset.
When I run 'fc-match sans', the output always will be DejaVu Sans. But, according to '60-latin-free.conf' - the first preferred sans-serif font is Noto Sans (which of course is installed on my system).
That problem also reproduces with serif (it's always DejaVu Serif) but not the first preferred - Heuristica.

To sum up, I have to edit '35-repl-custom.font' to set up proper font family.
Is that OK or something  wrong? (I use clean fontconfig)

Thanks in advance!
